scylla_docker.py

```diff
@@ -42,9 +42,15 @@ def _cmd(cmd, timeout=10, sudo=False):
             raise DockerCommandError('command: {}, error: {}, output: {}'.format(cmd, res.stderr, res.stdout))
         return res.stdout
 
+    def clean_old_images(self):
+        images = self._cmd('images -f "dangling=true" -q')
```

Comment:
does it work if multiple images are returned?
need to replace '\n' to space here?